ui/apps/platform/src/Containers/ComplianceEnhanced/Dashboard/Widgets/ComplianceByCluster.tsx

```diff
@@ -0,0 +1,134 @@
+import React, { useState } from 'react';
+import { Chart, ChartAxis, ChartBar, ChartGroup, ChartLabelProps } from '@patternfly/react-charts';
+
+import { LinkableChartLabel } from 'Components/PatternFly/Charts/LinkableChartLabel';
+import useResizeObserver from 'hooks/useResizeObserver';
+import {
+    defaultChartHeight,
+    defaultChartBarWidth,
+    patternflySeverityTheme,
+} from 'utils/chartUtils';
+
+import {
+    LOW_SEVERITY_COLOR,
+    MODERATE_MEDIUM_SEVERITY_COLOR,
+    IMPORTANT_HIGH_SEVERITY_COLOR,
+    CRITICAL_SEVERITY_COLOR,
+} from 'constants/visuals/colors';
+
+import WidgetCard from '../../../Dashboard/Widgets/WidgetCard';
```

Comment:
Any objections to pulling this out of the Dashboard container?

Reply:
Dave says to go for it. Can you do that as a follow up pull request?
Move WidgetCardEmptyState also at the same time